ajax/libs/selectonic/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "selectonic",
+  "version": "0.6.3",
+  "description": "jQuery-plugin for making any list of items selectable by mouse and keyboard.",
+  "author": {
+    "name": "Alexey Novichkov",
+    "email": "mygrib@gmail.com"
+  },
```

Comment:
How about to add homepage property as:

"homepage": "https://anovi.github.io/selectonic/"

Reply:
Sorry, I missed it!!